### DISCUSSION TITLE
kubernetes deployment? kubectl or helm chart

### DISCUSSION THREAD

**Discussion by encryptblockr** [thinkst/canarytokens-docker#146] [Feature Requests]
any plan to have yaml or helm chart to deploy this on kubernetes?

**Comment by jayjb** [comment]
Hi @encryptblockr,

We don't currently have it on our stack; but if you would be willing to guide us with a PR or some paths forward, I could potentially take a look.

**Comment by sabelomaga** [comment]
I've managed to deploy Canarytokens on Kubernetes using [Komopose](https://kompose.io/) to convert docker-compose.yml file to Kubernetes deployment (.yaml) files. I hope this helps. 🙂

**Comment by stone-z** [comment]
Hi folks :wave: I put together a `canarytokens` helm chart over at https://github.com/stone-z/canarytokens-k8s. It is very much a work in progress, but hopefully someone will find it useful
